app/src/components/Translations.vue

```diff
@@ -0,0 +1,53 @@
+<script>
+  var Vue = require('vue')
+  var VueI18n = require('vue-i18n')
+
+  // ready translated locales
+  var locales = {
+    de: {
```

Comment:
Jap, das könnte man so machen:

https://github.com/kazupon/vue-i18n/wiki/Usage-for-v3.0#dynamic-locale